janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -111,14 +111,14 @@
 
     private static final int DEFAULT_PORT = 9042;
 
+    private static Cluster cluster;
+    private static Map<String,Session> sessions = new ConcurrentHashMap<>();
```

Comment:
@amcp No it might not be a problem in testing. Do you think it might cause issues to use concurrent data structure and/or synchronized modifier here?